users/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db       import models
+
+from core.models     import TimeStampModel
+from programs.models import Program
+
+class User(TimeStampModel):
+    nickname          = models.CharField(max_length=100)
+    email             = models.CharField(max_length=300, unique=True)
+    profile_image_url = models.CharField(max_length=2000)
+    host              = models.BooleanField(default=False)
```

Comment:
boolean 값으로 저장하는 경우는 is_host라는 컬럼명을 사용해주세요~
```suggestion
    is_host              = models.BooleanField(default=False)
```